src/ScrollMath.js

```diff
@@ -464,17 +462,14 @@ module.exports = kind(
 			y0 = this.y;
 			x0 = this.x;
 		});
-		this.job = animation.requestAnimationFrame(fn);
+		fn();
```

Comment:
If we're going to invoke this immediately why don't we save a bindSafely call and just remove the wrapping function?